CHANGES.md

```diff
@@ -129,7 +133,6 @@
   + Add an option `--margin-check` to emit a warning if the formatted output exceeds the margin (#1110) (Guillaume Petiot)
   + Preserve comment indentation when `wrap-comments` is unset (#1138, #1159) (Jules Aguillon)
   + Improve error messages (#1147) (Jules Aguillon)
-  + Display standard output in the emacs plugin even when ocamlformat does not fail (#1189) (Guillaume Petiot)
```

Comment:
I'm not sure we should remove entries from the changelog unless the code has been removed in the same release. Here it can be useful for people running ocamlformat 0.13.0.

Reply:
Oh yes, sorry, that was not intentional, just what happened with `git revert`. I'll fix.